### PR TITLE
ci: Fix golangci lint checks

### PR DESCRIPTION
**Pull request by charmik-redhat**
## Checklist
- [ ] Investigated and inspected CI test results
- ~[ ] Unit test and regression tests added~
- ~[ ] Evaluated and added CHANGELOG entry if required~
- ~[ ] Determined and documented upgrade steps~
- ~[ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

If any of these don't apply, please comment below.

### Here I tell how I validated my change

CI is enough

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

**Review by dhaus67 — APPROVED** [review]
Pretty sure test files are skipped But let's be consistent with the comment style here ad well.

File: migrator/migrations/m_192_to_m_193_report_config_migrations/migration_test.go [hunk 1]
```diff
@@ -92,23 +92,23 @@ func (s *migrationTestSuite) TestMigration() {
 	}
 
 	//verify getMigratedReportConfigIfExists function
```
Comment (dhaus67) [inline migrator/migrations/m_192_to_m_193_report_config_migrations/migration_test.go:94]: Pretty sure test files are skipped but let's be consistent with comment style.

File: migrator/migrations/m_192_to_m_193_report_config_migrations/migration_test.go [hunk 2]
```diff
@@ -92,23 +92,23 @@ func (s *migrationTestSuite) TestMigration() {
...
 	s.NotNil(conf)
 
 	notifierID := conf.GetEmailConfig().GetNotifierId()
 
-	found, conf, err = getMigratedReportConfigIfExists("does-not-exist", dbs.GormDB, dbs.DBCtx)
+	found, conf, err = getMigratedReportConfigIfExists(dbs.DBCtx, "does-not-exist", dbs.GormDB)
 	s.Require().NoError(err)
 	s.False(found)
 	s.Nil(conf)
 	//verify checkifNotifierExists function
```
Comment (dhaus67) [inline migrator/migrations/m_192_to_m_193_report_config_migrations/migration_test.go:106]: Same as above.

**Review by dhaus67 — APPROVED** [review]

**Review by theencee — APPROVED** [review]
I wonder why this wasn't caught by CI before it was merged

**Comment by charmik-redhat** [comment]
> I wonder why this wasn't caught by CI before it was merged

The golangci lint wasn't showing any failure logs today and failing for everyone.

File: migrator/migrations/m_192_to_m_193_report_config_migrations/migration_test.go (hunk 1 above)
Comment (charmik-redhat) [reply migrator/migrations/m_192_to_m_193_report_config_migrations/migration_test.go]: Yeah updated them just to be safe

**Comment by theencee** [comment]
Enabled auto merge

File: migrator/migrations/m_192_to_m_193_report_config_migrations/migration_test.go (hunk 1 above)
Comment (theencee) [reply migrator/migrations/m_192_to_m_193_report_config_migrations/migration_test.go]: Looks like tests are evaluated
```
migrator/migrations/m_192_to_m_193_report_config_migrations/migration_test.go:118:31: rows.Err must be checked (rowserrcheck)
	configs, err := s.gormDB.Rows()
	                             ^
migrator/migrations/m_192_to_m_193_report_config_migrations/migration_test.go:120:41: rows.Err must be checked (rowserrcheck)
	snapshots, err := s.snapshotgormdB.Rows()
	                                       ^
```

**Pull request merged**